File: python/2/12.py
import pandas as pd

filename = "hightemp.txt"
df = pd.read_csv(filename, header=None, delimiter='\t')
df.iloc[:,0].to_csv('../../output/col1.txt', index=False, header=False)
df.iloc[:,1].to_csv('../../output/col2.txt', index=False, header=False)

File: ruby/2/12.rb
require "csv"

filename = "../../data/hightemp.txt"
table = CSV.table(filename, col_sep: "\t", headers: %w(pref city temp day))

File.open("col1.txt", "w") do |file|
  file.puts table[:pref]
end

File.open("col2.txt", "w") do |file|
  file.puts table[:city]
end

# %w(hoge fuga) => ["hoge", "fuga"]
